### DISCUSSION TITLE
Possible to have responsive 'mv-popup' ?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#896] [Q&A]
Hi

Is there a way to make 'mv-popup' responsive? (specially the width)

On mobile devices, if property is too much in the right side of the screen, the content of mv-popup will not be fully visible

**Comment by DmitrySharabin** [comment]
Have you tried re-define some of these styles? That might help.

https://github.com/mavoweb/mavo/blob/05a038c639a96ac677e488ed2c32a2f664ea4211/src-css/mavo.scss#L343-L426

**Comment by GalinhaLX** [comment]
The case I have, is that I can have images every where in the page 

User can add images like in a porfolio gallery

In case of an image in the right side of the viewport, if i change CSS styles to adapt to that case, then I have an issue for the images that are in the left side of viewport
(same for images on top or bottom)

I saw that [popper.js ](https://popper.js.org/)solves that problem, switching position correctly and automaticly, but i can't understand how it does it ..seems not only with CSS

that's why I proposed this[ plugin idea](https://github.com/mavoweb/plugins/issues/109)
could propably solve many use cases
